velox/type/Variant.cpp

```diff
@@ -77,6 +77,42 @@ struct VariantEquality<TypeKind::DATE> {
   }
 };
 
+template <>
+struct VariantEquality<TypeKind::SHORT_DECIMAL> {
+  template <bool NullEqualsNull>
+  static bool equals(const variant& a, const variant& b) {
+    const auto lhs = a.value<TypeKind::SHORT_DECIMAL>();
+    const auto rhs = b.value<TypeKind::SHORT_DECIMAL>();
+    const auto lType = DECIMAL(lhs.precision, lhs.scale);
+    const auto rType = DECIMAL(rhs.precision, rhs.scale);
+    if (!lType->equivalent(*rType))
+      return false;
```

Comment:
Hi, we use ClangTidy internally which enforces that braces are used for all if statements, even these single line ones.  Plus it helps with readability and it's defensive agains accidental bugs.

Could you add braces here?